views/options.erb

```diff
@@ -0,0 +1,24 @@
+<h1> <%= @player_name %> here are your options! </h1>
+<style>
+  body {
+    background-color: azure;
+  }
+  h1 {
+    background-color: white;
+    }
+
+  </style>
+
+<body>
+<p>Click an option and press submit!</p>
+
+<form action="/names">
```

Comment:
It's common practise for forms sending data to the server to make to use a POST request